src/domains/wallet-app/99_main.tf

```diff
@@ -0,0 +1,48 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "= 2.99.0"
```

Comment:
better to start directly with terraform v3 (3.40.0)

Reply:
you can refer to https://github.com/pagopa/io-infra/pull/422/files

Reply:
ok 🚀